asv/machine.py

```diff
@@ -119,6 +119,11 @@ class Machine(object):
          including its speed and class.  For example, 'Intel(R)
          Core(TM) i5-2520M CPU @ 2.50GHz (4 cores)'."""),
 
+        ("ncores",
```

Comment:
`num_cpu`?

Reply:
Actually I like more num_cpu, but it felt weird to use underscores in command line options.